ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -49,208 +41,121 @@ const styles = reactStyles({
-    <h3 style={styles.sectionHeader}>Active Runtime Operations</h3>
-    <FlexRow style={{'alignItems': 'center'}}>
-      <span style={{'marginRight': '1rem'}}>
-        {operation} in progress
-      </span>
-      <Button
-          onClick={() => abortRuntimeOperationForWorkspace(workspaceNamespace)}
-          data-test-id='active-runtime-operation'
-      >
-        Cancel
-      </Button>
-    </FlexRow>
-  </React.Fragment>;
-};
-
 export interface Props {
-  runtimeOps: RuntimeOpsStore;
   workspace: WorkspaceData;
 }
 
-interface State {
-  // Whether the initial runtime load is still in progress.
-  loading: boolean;
-  // Whether there was an error in loading the runtime data.
-  error: boolean;
-  // The runtime. null if none exists, or if there was an error in loading the
-  // runtime.
-  runtime: Runtime|null;
-}
-
-export const RuntimePanel = fp.flow(withCurrentWorkspace(), withStore(runtimeOpsStore, 'runtimeOps'))(
-  class extends React.Component<Props, State> {
-    private aborter = new AbortController();
-
-    constructor(props: Props) {
-      super(props);
-      this.state = {
-        loading: true,
-        error: false,
-        runtime: null
-      };
-    }
-
-    async componentDidMount() {
-      // TODO(RW-5420): Centralize a runtimeStore.
-      let runtime = null;
-      let error = false;
-      try {
-        const promise = runtimeApi().getRuntime(this.props.workspace.namespace, {signal: this.aborter.signal});
-        updateRuntimeOpsStoreForWorkspaceNamespace(this.props.workspace.namespace, {
-          promise: promise,
-          operation: 'get',
-          aborter: this.aborter
-        });
-        runtime = await promise;
-      } catch (e) {
-        // 404 is expected if the runtime doesn't exist, represent this as a null
-        // runtime rather than an error mode.
-        if (e.status !== 404) {
-          error = true;
-        }
-      }
-      markRuntimeOperationCompleteForWorkspace(this.props.workspace.namespace);
-      this.setState({
-        runtime,
-        error,
-        loading: false
-      });
-    }
-
-    render() {
-      const {runtimeOps, workspace} = this.props;
-      const {loading, error, runtime} = this.state;
-
-      const activeRuntimeOp: RuntimeOperation = runtimeOps.opsByWorkspaceNamespace[workspace.namespace];
-
-      if (loading) {
-        return <Spinner style={{width: '100%', marginTop: '5rem'}}/>;
-      } else if (error) {
-        return <div>Error loading compute configuration</div>;
-      } else if (!runtime) {
-        // TODO(RW-5591): Create runtime page goes here.
-        return <React.Fragment>
-          <div>No runtime exists yet</div>
-          {activeRuntimeOp && <hr/>}
-          {activeRuntimeOp && <div>
-            <ActiveRuntimeOp operation={activeRuntimeOp.operation} workspaceNamespace={workspace.namespace}/>
-          </div>}
-        </React.Fragment>;
-      }
+export const RuntimePanel = withCurrentWorkspace()(({workspace}) => {
```

Comment:
We don't use this prop, and I could get rid of it, but it would just cause more merge conflicts for #4100 to deal with.